packages/utilities/README.md

```diff
@@ -9,9 +14,20 @@
 [![npm](https://img.shields.io/npm/v/@sapphire/utilities?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/utilities)
 [![Depfu](https://badges.depfu.com/badges/4a8bc579d3807f58b04ed647f20c3db0/count.svg)](https://depfu.com/github/sapphire-project/utilities?project_id=15204)
 
-## Installation
+</div>
+
+## Description
 
-**Then install `@sapphire/utilities`**
+In our day-to-day life we often have a need for some function or type augmentation and having to include that function in every repo is a huge drag. To solve this problem there are dozens upon dozens of packages on NPM, but we cannot maintain those in case of issues and a lot of them are written poorly or not optimized. So instead our solution is to provide @sapphire/utilities, which is the single package that should you need to cover a lot of needs for the day-to-day life.
```

Comment:
```suggestion
We often have a need for a function or type augmentation and having to include it in every repo is a huge drag. To solve this problem there are dozens upon dozens of packages on NPM, but we cannot maintain those in case of issues and a lot of them are poorly written or under-optimised. Our solution is to provide @sapphire/utilities, which is the only package you'll likely need to cover your day-to-day needs.
```